packages/wallets/src/components/WalletCard/WalletCard.tsx

```diff
@@ -38,7 +38,10 @@ const WalletCard: React.FC<TProps> = ({ balance, currency, iconSize = 'lg', isDe
                             {currency} Wallet
                         </WalletText>
                         {isLoading ? (
-                            <div className='wallets-skeleton wallets-card--balance-loader' />
+                            <div
+                                className='wallets-skeleton wallets-card--balance-loader'
+                                data-testid='dt_wallet-card-balance-loader'
```

Comment:
just to follow the documentation https://wikijs.deriv.cloud/en/Frontend/development-guidelines/test_guide
```suggestion
                                data-testid='dt_wallet_card_balance_loader'
```